app-backend/batch/src/main/scala/com/azavea/rf/batch/aoi/airflow/UpdateAOIProject.scala

```diff
@@ -0,0 +1,96 @@
+package com.azavea.rf.batch.aoi.airflow
+
+import com.azavea.rf.batch.Job
+import com.azavea.rf.database.tables._
+import com.azavea.rf.database.{Database => DB}
+import com.azavea.rf.database.query._
+import com.azavea.rf.datamodel._
+
+import cats.data._
+import cats.implicits._
+
+import java.sql.Timestamp
+import java.time.ZonedDateTime
+import java.util.UUID
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+case class UpdateAOIProject(projectId: UUID)(implicit val database: DB) extends Job {
+  val name = FindAOIProjects.name
+
+  import database.driver.api._
+
+  def run: Unit = {
+    logger.info(s"Updating Project $projectId AOI...")
+    val result = for {
+      user: User                   <- OptionT(Users.getUserById(airflowUser))
+      (project: Project, aoi: AOI) <- OptionT(Projects.getAOIProject(projectId, user))
+      scenes: Iterable[UUID]       <- OptionT({
+        val area = aoi.area
+        val queryParams = aoi.filters.as[CombinedSceneQueryParams].getOrElse(CombinedSceneQueryParams())
```

Comment:
It's required, but we don't do any validation on it. The failure case I'm worried about is:

1. (by magic) a user has an AOI project with invalid filters and no scenes (or a small number of scenes) added
2. the first update check happens
3. they see a notice on their project that they have 450k scenes to review to add to their project
4. they feel overwhelmed, burn down their entire city, and publicly cite the AOI logic as why they did it